src/components/CSS/navBar.module.scss

```diff
@@ -0,0 +1,10 @@
+li a {
```

Comment:
This is dangerous; as far as I understand it, it'll mean that *every* link inside a `li` element will be styled as it is defined here. Depending on whether this is what you want, there are a few possible solutions:

1. explicitly add a class name to the elements you want to style, and use that class name here instead of `li a` (use this if you want to style only links that are created within the `navBar` component, because you can easily add class names to those elements)
2. wrap this in a class name - `.navBar li a {...}` (use this if you want to style all links inside `li` elements inside the `navBar` component, even when those components are created elsewhere)
3. move this declaration to an app-wide `scss` file so that it's obvious that these styles apply everywhere, not just inside the `navBar`.

In general, it's less error-prone to use custom class names directly on the elements you want to style than depending on HTML element hierarchy. The more you depend on some specific nesting, the harder it is to keep the styling and the component creating code in sync.